components/list/demo/list-drag-and-drop.js

```diff
@@ -167,7 +167,7 @@ class ListDemoDragAndDrop extends LitElement {
 		await this.updateComplete;
 
 		if (e.detail.keyboardActive) {
-			requestAnimationFrame(() => {
+			setTimeout(() => {
```

Comment:
Animation frame wasn't consistently waiting long enough?

Reply:
Right, meant to put a comment on this - this is unrelated to the change I made and this demo is broken currently in `main`.  Was talking to Dave B about it and this was the fix he had locally and just hadn't committed it yet, so I offered to do it as part of my PR